src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -872,5 +872,223 @@
       "path": "type",
       "notEquals": "Microsoft.ClassicStorage/storageAccounts"
     }
+  },
+  {
+    "id": "TA-000032",
+    "name": "APIM.BypassCertificateThumbprintOrNameValidation",
+    "shortDescription": "API Management calls to API backends should not bypass certificate thumbprint or name validation",
+    "fullDescription": "API Management calls to API backends should not bypass certificate thumbprint or name validation",
+    "recommendation": "To improve security, API Management should validate the backend server certificate for all API calls. Enable SSL certificate thumbprint and name validation by setting the tls.validateCertificateName and tls.validateCertificateChain properties to `true`.",
+    "helpUri": "https://github.com/Azure/template-analyzer/blob/main/docs/built-in-bpa-rules.md/#ta-000032-api-management-calls-to-api-backends-should-not-bypass-certificate-thumbprint-or-name-validation",
+    "severity": 2,
+    "evaluation": {
+      "resourceType": "Microsoft.ApiManagement/service/backends",
+      "allOf": [
+        {
+          "path": "properties.tls.validateCertificateName",
+          "equals": true
+        },
+        {
+          "path": "properties.tls.validateCertificateChain",
+          "equals": true
+        }
+      ]
+    }
+  },
+  {
+    "id": "TA-000033",
+    "name": "APIM.DirectAPIManagementEndpointShouldNotBeEnabled",
+    "shortDescription": "API Management direct management endpoint should not be enabled",
+    "fullDescription": "The direct management REST API in Azure API Management bypasses Azure Resource Manager role-based access control, authorization, and throttling mechanisms, thus increasing the vulnerability of your service.",
+    "recommendation": "Set the properties.enabled field to `false` to ensure that direct access to the management REST API is disabled.",
```

Comment:
```suggestion
    "recommendation": "Set the properties.enabled field to false to ensure that direct access to the management REST API is disabled.",
```

Reply:
Hey Vera, in the previous PR comments, I was asked to put `` for property fields. I had updated based on the PR comments. Here's an example comment: https://github.com/Azure/template-analyzer/pull/316#discussion_r1093823805